azure-pipelines.yml

```diff
@@ -58,9 +59,10 @@ jobs:
       meson --version
       doxygen --version
       echo "PATH=$PATH"
-    displayName: Display tool versions
+      printenv
```

Comment:
Are we sure this won't leak anything?

Reply:
Nothing an arbitrary person cannot leak already by opening a PR.

Reply:
Hm, maybe for master builds. Let me check

Reply:
OK, we should be fine. 

- Azure Pipelines masks variables it thinks are sensitive, as seen with "SYSTEM_ENABLEACCESSTOKEN=SecretVariable"
- Secret variables are not automatically injected into the environment, only on request (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#secret-variables). We certainly shouldn't insert any of them into the global env block.
- The only job that really needs secrets (the deploy release to GH job) does it without secret variables.
- And in general, we shouldn't make build output available for anything with real secrets in it; it's too easy to sneak in code somewhere (e.g. in a script on pip) that dumps environment variables.